src/ol/layer/BaseVector.js

```diff
@@ -219,13 +221,4 @@ class BaseVectorLayer extends Layer {
 }
 
 
-/**
- * Return the associated {@link module:ol/source/Vector vectorsource} of the layer.
- * @function
- * @return {import("../source/Vector.js").default} Source.
- * @api
- */
-BaseVectorLayer.prototype.getSource;
```

Comment:
Do we still get API docs for `#getSource` when this is removed?

Reply:
no, the api is broken. I will look into this

Reply:
JSDoc doesn't support having a generic type in the extends tag.
I've tried to fix it with: https://github.com/openlayers/jsdoc-plugin-typescript/pull/4

Reply:
done, the jsdoc plugin will be updated in #9184